### PR TITLE
chore: update changelog for version 2.13.1

### PR DESCRIPTION
**Pull request by wantsui**
- [x] update changelog for version 2.13.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,35 @@
...
+
+- Code Security (IAST): Always report a telemetry log error when an IAST propagation error raises, regardless of whether the \_DD_IAST_DEBUG environment variable is enabled or not.
+
+- Code Security: fix potential memory leak on IAST exception handling.
+
+- Profiling: all files with platform-dependent code have had their filenames updated to reflect the platform they are for. This fixes issues where the wrong file would be used on a given platform.
+
+- profiling: enables endpoint profiling for stack v2, `DD_PROFILING_STACK_V2_ENABLED` is set.
+
+- Fixes endpoint profiling when using libdatadog exporter, either with `DD_PROFILING_EXPORT_LIBDD_ENABLED` or `DD_PROFILING_TIMELINE_ENABLED`.
```
Comment (taegyunkim) [inline CHANGELOG.md:26]:
```suggestion
- profiling: fixes endpoint profiling when using libdatadog exporter, either with `DD_PROFILING_EXPORT_LIBDD_ENABLED` or `DD_PROFILING_TIMELINE_ENABLED`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,29 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
 
 ---
 
+## 2.13.1
+
+
+### Bug Fixes
+
+- Code Security (IAST)
+  - Always report a telemetry log error when an IAST propagation error raises, regardless of whether the \_DD_IAST_DEBUG environment variable is enabled or not.
```
Comment (erikayasuda) [inline CHANGELOG.md:13]:
```suggestion
  - Always report a telemetry log error when an IAST propagation error raises, regardless of whether the `_DD_IAST_DEBUG` environment variable is enabled or not.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,29 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
...
+  - Code Security: fix potential memory leak on IAST exception handling.
```
Comment (erikayasuda) [inline CHANGELOG.md:14]:
```suggestion
  - Code Security: Fixes potential memory leak on IAST exception handling.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,29 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
...
+
+- Profiling:
+  - All files with platform-dependent code have had their filenames updated to reflect the platform they are for. This fixes issues where the wrong file would be used on a given platform.
```
Comment (erikayasuda) [inline CHANGELOG.md:17]:
```suggestion
  - Updates filenames for all files with platform-dependent code to reflect the platform they are for. This fixes issues where the wrong file would be used on a given platform.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,29 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
...
+  - Enables endpoint profiling for stack v2, `DD_PROFILING_STACK_V2_ENABLED` is set.
+  - Fixes endpoint profiling when using libdatadog exporter, either with `DD_PROFILING_EXPORT_LIBDD_ENABLED` or `DD_PROFILING_TIMELINE_ENABLED`.
+  - Enables code provenance when using libdatadog exporter, `DD_PROFILING_EXPORT_LIBDD_ENABLED`, `DD_PROFILING_STACK_V2_ENABLED`, or `DD_PROFILING_TIMELINE_ENABLED`.
+  - Fixes an issue where flamegraph was upside down for stack v2,  `DD_PROFILING_STACK_V2_ENABLED`.
```
Comment (erikayasuda) [inline CHANGELOG.md:21]:
```suggestion
  - Fixes an issue where the flamegraph was upside down for stack v2 when enabling  `DD_PROFILING_STACK_V2_ENABLED`.
```

**Review by erikayasuda — APPROVED** [review]
Just some nits! Thanks! 😄

**Pull request merged**